documentation/src/main/asciidoc/user_guide/integrations.adoc

```diff
@@ -1227,6 +1227,12 @@ $$3.$$ Configure persistence.xml to use Infinispan instead of JBoss Cache:
 
 ----
 
+===  Using Infinispan CDI integration
+
+Infinispan supports both injecting remote and embedded Caches as well as bridging events at Cache and CacheManager level.
+
+For more information, have a look at our link:../user_guide/user_guide.html#_cdi_support[CDI paragraph].
```

Comment:
the underscore prefix on paragraph anchors is a relic of the past. Also since this is an anchor in this same document, you can use link:#cdi_support[CDI paragraph].